docs/install/rpm.md

```diff
@@ -38,10 +41,10 @@ Installing and running Open Distro from an RPM package is a more manual process
    sudo yum list opendistroforelasticsearch --showduplicates
    ```
 
-1. Choose the version you'd like and install it:
+1. Choose the version you'd like and install it: (See [note](#rpm-package) above.)
```

Comment:
Should this be "Choose the version you'd like to install:"?

Reply:
Yes, that's better - changed based on the suggestion